### ISSUE TITLE
PHP v8 compatibility

### ISSUE THREAD

**Issue by marcoherzog** [OpenSchulportfolio/openschulportfolio-build#10]
Are there any plans to make the software compatible with php v8?

**Comment by ironiemix** [comment]
Yes. As soon as the underlying DokuWiki is fully compatible with PHP8 I will try to create a OSP release that will be too.

https://github.com/splitbrain/dokuwiki/issues?q=is%3Aissue+is%3Aopen+label%3A%22PHP+8%22

**Comment by ricki-z** [comment]
Hello,
seems like DokuWiki is now compatible with PHP 8.0 and 8.1:
https://www.dokuwiki.org/requirements
Will OpenSchulPortfolio also be compatible to PHP 8.1 with that version of DokuWiki?

**Comment by dasJ** [comment]
[Since there is no supported PHP 7 anymore](https://endoflife.date/php), will there be any update on this?

**Comment by dasJ** [comment]
Clearly no
